rosflight/src/rosflight_io.cpp

```diff
@@ -404,7 +406,10 @@ void rosflightIO::handle_attitude_quaternion_msg(const mavlink_message_t &msg)
   mavlink_msg_attitude_quaternion_decode(&msg, &attitude);
 
   rosflight_msgs::Attitude attitude_msg;
-  attitude_msg.header.stamp = mavrosflight_->time.get_ros_time_ms(attitude.time_boot_ms);
+  ros::Time now;
+  std::chrono::milliseconds time_boot(attitude.time_boot_ms);
+  now.fromNSec(mavrosflight_->time.get_time_boot(std::chrono::nanoseconds(time_boot)).count());
+  attitude_msg.header.stamp = now;
```

Comment:
These (nearly the) same lines seem to get repeated frequently. What if we add this function to `RosflightIO`:
```cpp
ros::Time fcu_time_to_ros_time(std::chrono::nanoseconds fcu_time)
{
  ros::Time now;
  now.fromNSec(mavrosflight_->time.fcu_time_to_system_time(fcu_time).count());
  return now;
}
```
Then it could be called as
```cpp
msg.header.stamp = fcu_time_to_ros_time(std::chrono::milliseconds(boot_ms));
```
or
```cpp
msg.header.stamp = fcu_time_to_ros_time(std::chrono::microseconds(boot_us));
```